examples/js/loaders/SVGLoader.js

```diff
@@ -683,6 +726,298 @@ THREE.SVGLoader.prototype = {
+
+							// Angle
+							angle = array[ 0 ] * Math.PI / 180;
+
+							if ( array.length >= 3 ) {
+
+								// Center x, y
+								cx = array[ 1 ];
+								cy = array[ 2 ];
+
+							}
+
+							// Rotate around center (cx, cy)
+
+							var translation = createIdTransform();
+							translation[ 4 ] = -cx;
+							translation[ 5 ] = -cy;
+
+							var rotation = createRotationTransform( angle );
+
+							var translRot = createIdTransform();
+							multiplyTransforms( translation, rotation, translRot );
+
+							translation[ 4 ] = cx;
+							translation[ 5 ] = cy;
+							multiplyTransforms( translRot, translation, transform );
+
+						}
+
+						break;
+
+					case "scale":
+
+						if ( array.length >= 1 ) {
+
+							transform = createIdTransform();
+
+							var scaleX = array[ 0 ];
+							var scaleY = scaleX;
+
+							if ( array.length >= 2 ) {
+								scaleY = array[ 1 ];
+							}
+
+							transform[ 0 ] = scaleX;
+							transform[ 3 ] = scaleY;
+
+						}
+
+						break;
+
+					case "skewX":
+
+						if ( array.length === 1 ) {
+
+							transform = createIdTransform();
+
+							transform[ 2 ] = Math.tan( array[ 0 ] * Math.PI / 180 );
+
+						}
+
+						break;
+
+					case "skewY":
+
+						if ( array.length === 1 ) {
+
+							transform = createIdTransform();
+
+							transform[ 1 ] = Math.tan( array[ 0 ] * Math.PI / 180 );
+
+						}
+
+						break;
+
+					case "matrix":
+
+						if ( array.length === 6 ) {
+
+							transform = array;
+
+						}
+
+						break;
+				}
+
+			}
+			
+			return transform;
+
+		}
+
+		function transformPath( path, m ) {
+
+			var isRotated = isTransformRotated( m );
+
+			function transfVec2( v ) {
+
+				var x = v.x * m[ 0 ] + v.y * m[ 2 ] + m[ 4 ];
+				var y = v.x * m[ 1 ] + v.y * m[ 3 ] + m[ 5 ];
+
+				v.set( x, y );
+
+			}
+
+			var subPaths = path.subPaths;
+
+			for ( var i = 0, n = subPaths.length; i < n; i++ ) {
+
+				var subPath = subPaths[ i ];
+				var curves = subPath.curves;
+
+				for ( var j = 0; j < curves.length; j++ ) {
+
+					var curve = curves[ j ];
+
+					if ( curve.isLineCurve ) {
+
+						transfVec2( curve.v1 );
+						transfVec2( curve.v2 );
+
+					}
+					else if ( curve.isCubicBezierCurve ) {
+
+						transfVec2( curve.v0 );
+						transfVec2( curve.v1 );
+						transfVec2( curve.v2 );
+						transfVec2( curve.v3 );
+
+					}
+					else if ( curve.isQuadraticBezierCurve ) {
+
+						transfVec2( curve.v0 );
+						transfVec2( curve.v1 );
+						transfVec2( curve.v2 );
+
+					}
+					else if ( curve.isEllipseCurve ) {
+
+						if ( isRotated ) {
+							console.warn( "SVGLoader: Elliptic arc or ellipse rotation or skewing is not implemented." );
+						}
+
+						curve.xRadius *= getTransformScaleX( m );
+						curve.yRadius *= getTransformScaleY( m );
+
+					}
+
+				}
+	
+			}
+
+		}
+
+		function createIdTransform( m ) {
+
+			// 2 x 3 matrix:
+			// 1  0
+			// 0  1
+			// tx ty
+
+			if ( ! m ) {
+				m = [];
+			}
+
+			m[ 0 ] = 1;
```

Comment:
How about using `THREE.Matrix3` instead?

Reply:
I thought of it but decided the other way. Ok, I'm doing the change right now.